<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="LAPTOP-AIO68RGA" ignored="0" name="注册接口" tests="9" failures="7" timestamp="2021-09-25T23:28:10 CST" time="0.553" errors="0">
  <testcase name="loginFailed" time="0.143" classname="com.lemon.LoginTest">
    <failure type="java.lang.AssertionError" message="登录失败1">
      <![CDATA[java.lang.AssertionError: 登录失败1
at com.lemon.LoginTest.loginFailed(LoginTest.java:42)
at java.util.ArrayList.forEach(ArrayList.java:1249)
... Removed 32 stack frames]]>
    </failure>
  </testcase> <!-- loginFailed -->
  <testcase name="loginFailed2" time="0.002" classname="com.lemon.LoginTest">
    <failure type="java.lang.AssertionError" message="登录失败2">
      <![CDATA[java.lang.AssertionError: 登录失败2
at com.lemon.LoginTest.loginFailed2(LoginTest.java:47)
at java.util.ArrayList.forEach(ArrayList.java:1249)
... Removed 32 stack frames]]>
    </failure>
  </testcase> <!-- loginFailed2 -->
  <testcase name="loginFailed3" time="0.002" classname="com.lemon.LoginTest">
    <failure type="java.lang.AssertionError" message="登录失败3">
      <![CDATA[java.lang.AssertionError: 登录失败3
at com.lemon.LoginTest.loginFailed3(LoginTest.java:52)
at java.util.ArrayList.forEach(ArrayList.java:1249)
... Removed 32 stack frames]]>
    </failure>
  </testcase> <!-- loginFailed3 -->
  <testcase name="loginSuccess" time="0.003" classname="com.lemon.LoginTest"/>
  <testcase name="test" time="0.002" classname="com.lemon.LoginTest">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
at com.lemon.LoginTest.test(LoginTest.java:30)
at java.util.ArrayList.forEach(ArrayList.java:1249)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- test -->
  <testcase name="registerFailed" time="0.002" classname="com.lemon.RegisterTest">
    <failure type="java.lang.AssertionError" message="注册失败1">
      <![CDATA[java.lang.AssertionError: 注册失败1
at com.lemon.RegisterTest.registerFailed(RegisterTest.java:28)
at java.util.ArrayList.forEach(ArrayList.java:1249)
... Removed 32 stack frames]]>
    </failure>
  </testcase> <!-- registerFailed -->
  <testcase name="registerFailed2" time="0.002" classname="com.lemon.RegisterTest">
    <failure type="java.lang.AssertionError" message="注册失败2">
      <![CDATA[java.lang.AssertionError: 注册失败2
at com.lemon.RegisterTest.registerFailed2(RegisterTest.java:33)
at java.util.ArrayList.forEach(ArrayList.java:1249)
... Removed 32 stack frames]]>
    </failure>
  </testcase> <!-- registerFailed2 -->
  <testcase name="registerFailed3" time="0.004" classname="com.lemon.RegisterTest">
    <failure type="java.lang.AssertionError" message="注册失败3">
      <![CDATA[java.lang.AssertionError: 注册失败3
at com.lemon.RegisterTest.registerFailed3(RegisterTest.java:38)
at java.util.ArrayList.forEach(ArrayList.java:1249)
... Removed 32 stack frames]]>
    </failure>
  </testcase> <!-- registerFailed3 -->
  <testcase name="registerSuccess" time="0.002" classname="com.lemon.RegisterTest"/>
</testsuite> <!-- 注册接口 -->
